resources/scripts/dev/cheat_command_overlay.lua

```diff
@@ -11,21 +12,21 @@ local CheatOverlay = {}
 ---@class CheatCommandEntry
 ---@field command string
 ---@field label string
----@field draw fun(entry: CheatCommandEntry, ctx:NuklearContext) : boolean
+---@field draw fun(entry: CheatCommandEntry) : boolean
 
 ---@type table<string, CheatCommandEntry>
 local availableCommands = {}
 
-local enabledColor = { 50, 100, 50, 255 }
-local disabledColor = { 100, 50, 50, 255 }
-local enabledHoveredColor = { 25, 75, 25, 255 }
-local disabledHoveredColor = { 75, 25, 25, 255 }
+local enabledColor = Utilities.color(0.19, 0.39, 0.19, 1)
+local disabledColor = Utilities.color(0.39, 0.19, 0.19, 1)
+local enabledHoveredColor = Utilities.color(0.09, 0.29, 0.09, 1)
+local disabledHoveredColor = Utilities.color(0.29, 0.09, 0.09, 1)
+local buttonSize = { w = -0.001, h = 30 }
```

Comment:
Why `-0.001`? X)

Reply:
Mmh 🤔 This branch has been stale for a long time. I wonder if I did it because ImGui want something like that. I'll try to play with it and come back with more info 😆

Reply:
Ok, yes of course. In general negative size in ImGui is treated as "Fill". I changed it to be just `-1` instead. It feels less random. This is mostly an ImGui common pattern so I'm not sure if I should add a comment to it.